libs/designsystem/src/lib/components/item/item.component.ts

```diff
@@ -13,12 +19,16 @@ export enum ItemSize {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ItemComponent {
+  _selectable: boolean;
+
   @Input() disabled: boolean;
   @HostBinding('class.selected')
   @Input()
   selected: boolean;
-  @Input()
-  selectable: boolean;
+  @Input() set selectable(selectable: boolean) {
+    this._selectable = selectable;
+    this.cdr.markForCheck();
```

Comment:
```suggestion
    if (this._selectable !== selectable) {
      this._selectable = selectable;
      this.cdr.markForCheck();
    }
```

Reply:
Consider comparing simple types before dirty marking

Reply:
Implemented (a slight variation) in 13b9fef2de072f367d733deabe27c63b35ec054a